language/stdlib/modules/Errors.move

```diff
@@ -63,70 +65,70 @@ module Errors {
     public fun invalid_state(reason: u64): u64 { make(INVALID_STATE, reason) }
     spec fun invalid_state {
         pragma opaque = true;
-        aborts_if false;
+        aborts_if INVALID_STATE + (reason << 8) > max_u64();
```

Comment:
Same here. All those should be aborts_if false for the caller. Otherwise the prover has to show all the time this does not happen. (It doesn't because of constants passed to those functions, but prover has to show this.)